prow/plugins/config.go

```diff
@@ -612,6 +627,18 @@ func (c *Configuration) TriggerFor(org, repo string) Trigger {
 	return Trigger{}
 }
 
+// DcoFor finds the Dco for a repo, if one exists
+// a Dco can be listed for the repo itself or for the
+// owning organization
+func (c *Configuration) DcoFor(org, repo string) Dco {
```

Comment:
This approach does not really allow for repo-level overrides of org-level fields. I think I linked to #9757 before -- do we care?

Reply:
(you can get org-level overrides or repo-level overrides with appropriate ordering, but that's not obvious)

Reply:
@stevekuznetsov I've looked into this and I think I better understand how it works. If this is something we want, I can give it a try. Do we want to give it a try or wait for a library to be implemented?

Reply:
>  or wait for a library to be implemented?
We need generics for that :S

Reply:
There's an attempt to codegen it currently WIPO